cfpb/consumerfinance.gov
# Add rule to remove trailing . when visiting the site

Currently we don't redirect back to the consumerfinance.gov when a user submits consumerfinance.gov. 

This rule should strip the trailing . from the request per this [stack overflow post](https://stackoverflow.com/questions/31972945/how-to-remove-the-trailing-dot-after-tld-by-using-mod-rewrite)


---




## Additions

- Remove trailing . from requests.

## How to test this PR

1. Visit the deployed docker stack with a trailing . on the url and confirm you are redirected.


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
